src/main/webapp/WEB-INF/templates/landing.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<!-- Normally this will be a site to accept a user name and password and log in to determine which code to send, for now it'll just redirect-->
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Inferno Reference Server</title>
+  <meta name="description" content="Authorize Cindehr">
+  
+  <link href="../js/lib/bootstrap-4.5.3-dist/css/bootstrap.min.css" rel="stylesheet"> 
+  <link href="../css/site.css" rel="stylesheet">
+</head>
+
+<body>
+	<div th:replace="tmpl-banner :: banner"></div>
+	
+	<br>
+	<br>
+	
+    <div id="pageContent" class="container">
+   		
+   		<p>This server is designed to meet the Inferno Program Certification Tests.  This includes:</p>
+
+<p>Support for SMART-on-FHIR, including:</p>
+		<ul>
+			<li>Public and Confidential Clients</li>
+			<li>Standalone and EHR Launch</li>
+			<li>Token Refresh</li>
+			<li>Token Revocation</li>
+		</ul>
+
+<p>Support for US Core v3.1.0, including:</p>
```

Comment:
Should we bump this to `v3.1.1`, since we're about to release 3.1.1 support?
```suggestion
<p>Support for US Core v3.1.1, including:</p>
```